py/makeversionhdr.py

```diff
@@ -57,7 +57,7 @@ def get_version_info_from_git():
     return git_tag, git_hash, ver
 
 def get_version_info_from_docs_conf():
-    with open("%s/docs/conf.py" % sys.argv[0].rsplit("/", 2)[0]) as f:
+    with open(os.path.join(os.path.dirname(sys.argv[0]), "docs", "conf.py")) as f:
```

Comment:
You are correct, somehow I thought this worked since the build succeeded. How come `Build -> Clean` did not get rid of the generated files?